routes/api.php

```diff
@@ -17,6 +18,7 @@
 |
 */
 Route::middleware('auth:sanctum')->group(function () {
+    Route::get('users', [UserController::class, 'index']);
```

Comment:
```suggestion
  Route::get('users', [UserController::class, 'index'])->middleware(['ability:get-users']);
```
Só precisa aguardar o marcel fazer alguns ajustes que solicitei e então poderá alterar o código e inserir essa permissão na rota. Já testei e funcionou corretamente.